apps/kyberswap-interface/src/pages/Earns/constants.ts

```diff
@@ -188,5 +202,22 @@ export const PROTOCOLS_CORE_MAPPING: Record<EarnDex, CoreProtocol> = {
   [EarnDex.DEX_THENAFUSION]: CoreProtocol.AlgebraV1,
   [EarnDex.DEX_KODIAK_V3]: CoreProtocol.UniswapV3,
   [EarnDex.DEX_UNISWAPV2]: CoreProtocol.UniswapV2,
-  // [EarnDex.DEX_UNISWAP_V4]: CoreProtocol.UniswapV4,
+  [EarnDex.DEX_UNISWAP_V4]: CoreProtocol.UniswapV4,
+}
+
+export const UNISWAPV4_STATEVIEW_CONTRACT: Record<EarnChain, string> = {
+  [EarnChain.MAINNET]: '0x7ffe42c4a5deea5b0fec41c94c136cf115597227',
+  [EarnChain.BASE]: '0xa3c0c9b65bad0b08107aa264b0f3db444b867a71',
+  [EarnChain.BSC]: '0xd13dd3d6e93f276fafc9db9e6bb47c1180aee0c4',
+  [EarnChain.ARBITRUM]: '0x76fd297e2d437cd7f76d50f01afe6160f86e9990',
+  [EarnChain.AVAX]: '0xc3c9e198c735a4b97e3e683f391ccbdd60b69286',
+  [EarnChain.OPTIMISM]: '0xc18a3169788f4f75a170290584eca6395c75ecdb',
+  [EarnChain.MATIC]: '0x5ea1bd7974c8a611cbab0bdcafcb1d9cc9b3ba5a',
+  [EarnChain.BERA]: '',
```

Comment:
do we miss the config for Bera?

Reply:
@viet-nv univ4 has not supported bera yet